doctrine/static/searchtools.js

```diff
@@ -0,0 +1,761 @@
+/*
```

Comment:
ARGH, can't we use an already existing version hosted somewhere?

Reply:
I have no clue, the other project I picked up randomly also has the `init` missing but somehow the search there is working... I think we can wait for their reply